{"name": "PayPal提现验证码错误", "status": "failed", "statusDetails": {"message": "AssertionError: assert '验证码错误。 请确保输入您收到的最新验证码。' == 'Verification...you received.'\n  - Verification Code Error. Please make sure to enter the latest verification code you received.\n  + 验证码错误。 请确保输入您收到的最新验证码。", "trace": "drivers = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"a21c3529fe3a66a47369d1d8b66c8024\")>\n\n    @compose(feature=\"提现页面\", story=\"提现\", title='PayPal提现验证码错误')\n    @pytest.mark.Withdraw_Money\n    def test_withdraw_money_success(drivers):\n        withdraw = WithdrawPageOperation(drivers)\n        login = LoginPageOperation(drivers)\n        with allure.step(f\"手动验证登录开始！\"):\n            withdraw.open_url('https://www.coupert.com/user/login')\n            email = 'burtontest@test.com'\n            password = 123456\n            login.input_email(email)\n            login.input_password(password)\n            login.visible_account_email()\n            time.sleep(5)\n        with allure.step('开始PayPal提现'):\n            withdraw.open_url('https://www.coupert.com/user/withdraw')\n            # 进入提现页面\n            withdraw.entry_withdraw_process()\n            # 选择Paypal提现\n            withdraw.select_paypal()\n            # paypal账号勾选确认\n            withdraw.click_confirm_check(email)\n            # 进入下一步\n            withdraw.click_next_button()\n            # 发送验证码\n            withdraw.click_verify_code_button()\n            # 点击发送验证码\n            withdraw.click_verify_code_button()\n            # 数据库获取验证码\n            ...\n            # 输入验证码\n            withdraw.input_verify_code(1234)\n            # 点击提现\n            withdraw.click_withdraw_button()\n            # 获取失败提示文案\n            error_text = withdraw.get_verify_wrong_text()\n            # 断言提现失败文案是否与预期一致\n>           assert error_text == AM.VERIFY_WRONG_MESSAGE\nE           AssertionError: assert '验证码错误。 请确保输入您收到的最新验证码。' == 'Verification...you received.'\nE             - Verification Code Error. Please make sure to enter the latest verification code you received.\nE             + 验证码错误。 请确保输入您收到的最新验证码。\n\nsrc/case/test_withdraw_giftcard.py:91: AssertionError"}, "steps": [{"name": "手动验证登录开始！", "status": "passed", "start": 1675137602460, "stop": 1675137636878}, {"name": "开始PayPal提现", "status": "failed", "statusDetails": {"message": "AssertionError: assert '验证码错误。 请确保输入您收到的最新验证码。' == 'Verification...you received.'\n  - Verification Code Error. Please make sure to enter the latest verification code you received.\n  + 验证码错误。 请确保输入您收到的最新验证码。\n", "trace": "  File \"/Users/soar/PycharmProjects/soar_uitest/src/case/test_withdraw_giftcard.py\", line 91, in test_withdraw_money_success\n    assert error_text == AM.VERIFY_WRONG_MESSAGE\n"}, "start": 1675137636879, "stop": 1675137672621}], "attachments": [{"name": "log", "source": "9492d881-8666-4e4c-8320-92299eb1ad00-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "d90f1b77-2777-4c78-8fa3-321981b7a26f-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "30a8e38d-f321-4b04-bb59-6bb5e31af430-attachment.txt", "type": "text/plain"}], "start": 1675137602459, "stop": 1675137672623, "uuid": "983bff77-bc4e-4280-8aed-7d5b5392af3d", "historyId": "665be149f4842510ac0ce2ae45bfae2c", "testCaseId": "5ee229eac13853cbe933ad0715db8cb7", "fullName": "src.case.test_withdraw_giftcard#test_withdraw_money_success", "labels": [{"name": "feature", "value": "提现页面"}, {"name": "story", "value": "提现"}, {"name": "tag", "value": "Withdraw_Money"}, {"name": "parentSuite", "value": "src.case"}, {"name": "suite", "value": "test_withdraw_giftcard"}, {"name": "host", "value": "soardeMac-mini-9.local"}, {"name": "thread", "value": "89296-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.case.test_withdraw_giftcard"}]}